### ISSUE TITLE
the specific hyper-parameters for node classification

### ISSUE THREAD

**Issue by SiyuanHuangSJTU** [Wu-Junran/SEP#5]
Hi, I have read your paper and code and I find them very interesting and inspiring. However, when trying to follow up on your work on node classification, for the hyperparameter settings, you have only mentioned in your paper:

**For model configuration, the learning rate is set to 0.01, the hidden size is set ∈ {16, 32, 128, 256}, weight decay is set ∈ {0.02, 5 × 10−4}, and dropout rate for each layer is set ∈ {0, . . . , 0.9}.**

I didn't find in your paper and code the specific hyperparameters for Cora Citeseer and Pubmed. Could you publish them? It would help us to reproduce and follow up your work, thank you very much!
